interface/patient_file/summary/add_edit_issue.php

```diff
@@ -375,6 +375,7 @@ function ActiveIssueCodeRecycleFn($thispid2, $ISSUE_TYPES2) {
 <html>
 <head>
 <?php html_header_show();?>
+    <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-9-1/index.js"></script>
```

Comment:
per above:
I am suggesting to move it 20 lines down(to between line 399 and line 400) to bundle it with the other javascript library includes. This is to simply avoid another developer coming along in the future and placing another jquery library include which can really mess things up (note this has happened :) ).